vhdbuilder/packer/install-dependencies.sh

```diff
@@ -566,6 +566,12 @@ done
 rm -f ./azcopy # cleanup immediately after usage will return in two downloads
 stop_watch $capture_time "Download and Process Kubernetes Packages / Extract Binaries" false
 
+# We dont call out to apt after this in our build process, therefore restoring sources.list back to its original state if we used canonical snapshot earlier
+# This will also always happen when cloud-init is applied, therefore, all AKS nodes will always have a fresh sources.list regardless
+if [[ "${OS_VERSION}" == "22.04" && -n "${VHD_BUILD_TIMESTAMP}" ]]; then
```

Comment:
nit: `[ "${OS_VERSION}" == "22.04" ] && [  -n "${VHD_BUILD_TIMESTAMP}" ]`

Reply:
eh, we have this format elsewhere in the file so unless absolutely necessary, not changing

eg https://github.com/Azure/AgentBaker/blob/952c143d3dd7febe6dec46c283b84b6c81c32e57/vhdbuilder/packer/pre-install-dependencies.sh#L123

fwiw, our format isn't consistent at all for conditionals, we need a big refactor to align it everywhere

Reply:
we should really do that :)